chartfx-math/src/main/java/de/gsi/math/MathBaseGenBase.java

```diff
@@ -0,0 +1,429 @@
+     * Stefan-Boltzmann constant [W m^-2 K^-4]
+     */
+    public static final double SIGMA = 5.6704e-8;
+    /**
+     * Uncertainty of Stefan-Boltzmann constant
+     */
+    public static final double SIGMA_UNCERTAINTY = 0.000040e-8;
+    /**
+     * Avogadro constant (Avogadro's Number) [mol^-1]
+     */
+    public static final double N_A = 6.02214199e+23;
+    /**
+     * Uncertainty of the Avogadro constant
+     */
+    public static final double N_A_UNCERTAINTY = 0.00000047e+23;
+    /**
+     * universal gas constant (Na * K) [J K^-1 mol^-1]
+     * http://scienceworld.wolfram.com/physics/UniversalGasConstant.html
+     */
+    public static final double R = K * N_A;
+    /**
+     * Uncertainty of the Dry Air Gas Constant
+     */
+    public static final double H_UNCERTAINTY = 0.00000052e-34;
+    /**
+     * Dry Air Gas Constant (R / MWair) [erg s]
+     * http://atmos.nmsu.edu/education_and_outreach/encyclopedia/gas_constant.htm
+     */
+    public static final double H_CGS = 1.0e7 * H;
+    /**
+     * h_cgs * c_cgs [erg cm]
+     */
+    public static final double HC_CGS = H_CGS * C_CGS;
+    /**
+     * Molecular weight of dry air [kg kmol^-1 (or gm mol^-1)]
+     *  1976 US Standard Atmosphere,
+     *  also see http://atmos.nmsu.edu/jsdap/encyclopediawork.html
+     */
+    public static final double MW_AIR = 28.9644;
+    /**
+     *  [J kg^-1 K^-1]
+     */
+    public static final double RG_AIR = (1000.0 * R) / MW_AIR;
+    /**
+     *
+     */
+    public static final double R_UNCERTAINTY = R * ((K_UNCERTAINTY / K) + (N_A_UNCERTAINTY / N_A));
+
+    public static double log(double x) {
+        return java.lang.Math.log(x);
+    }
+
+    public static double log10(double x) {
+        return java.lang.Math.log10(x);
+    }
+
+    public static double aCosH(double arg0) {
+        if (arg0 == 0.0)
+            return 0.0;
+        double ax = java.lang.Math.abs(arg0);
+        return java.lang.Math.log(arg0 + ax * java.lang.Math.sqrt(1. - 1. / (ax * ax)));
+    }
+
+    public static double aSinH(double arg0) {
+        if (arg0 == 0.0)
+            return 0.0;
+        double ax = java.lang.Math.abs(arg0);
+        return java.lang.Math.log(arg0 + ax * java.lang.Math.sqrt(1. + 1. / (ax * ax)));
+    }
+
+    public static double aTanH(double arg0) {
+        return java.lang.Math.log((1 + arg0) / (1 - arg0)) / 2;
+    }
+
+    // codegen: double -> long
+    /**
+     * some integer math
+     *
+     * @param x input px
+     * @param y input py
+     * @return sqrt(px*px + py*py)
+     */
+    public static double hypot(double x, double y) {
+        return java.lang.Math.hypot(x, y);
+    }
+    //// end codegen
+
+    //// codegen: double -> float, int, long
+    public static double abs(double d) {
+        return (d >= 0) ? d : -d;
+        // short: return (d >= 0) ? (short) d : (short) (-d);
+    }
+    //// end codegen
+
+    public static double aCos(double arg0) {
+        return java.lang.Math.acos(arg0);
```

Comment:
Codacy found an issue: [Unnecessary use of fully qualified name 'java.lang.Math.acos' due to existing implicit import 'java.lang.*'](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5780327)